website/project/model.py

```diff
@@ -1519,11 +1551,7 @@ def register_node(self, schema, auth, template, data):
         :param auth: All the auth information including user, API key.
         :template: Template name
         :data: Form data
-
         """
-        if not self.can_edit(auth):
```

Comment:
We discussed this in person a few weeks ago and decided that it's the responsibility of the view to check authorization here. This is also consistent with other model methods. Are we changing policy on this?